std_msgs/Header header

float32 zoom		# zoom ratio [X]
byte mode 		# control mode {0/level,1/velocity}
